app/assets/stylesheets/forever_style_guide/modules/_pnav.scss

```diff
@@ -0,0 +1,464 @@
+  align-self: flex-start;
+  cursor: pointer;
+  border: none;
+  outline: none;
+
+  &::before,
+  &::after {
+    position: absolute;
+    right: 0;
+    width: 22px;
+    height: 2px;
+    content: ' ';
+    border-radius: 1px;
+    background-color: $color-gray-600;
+    @include transition_fast(background-color);
+  }
+
+  &::before { transform: rotate(45deg); }
+  &::after { transform: rotate(-45deg); }
+
+  &:hover {
+    &::before,
+    &::after { background-color: $color-forever_blue; }
+  }
+}
+
+.f-pnav__side-section-info {
+  padding-bottom: 15px;
+
+  /* avatar/badge and storage meter */
+  .f-pnav__side-info-avameter {
+    display: flex;
+    align-items: center;
+  }
+
+  .f-pnav__side-info-avatar {
+    position: relative;
+    flex: 0 1 auto;
+
+    /* necessary to dig into avatar and add custom size*/
+    .avatar {
+      width: 80px;
+      height: 80px;
+    }
+
+    img {
+      position: absolute;
+      height: 50px;
+      width: 50px;
+      top: 48px;
+      left: 43px;
+    }
+  }
+
+  .f-pnav__side-info-storage-meter {
+    flex: 1 0 auto;
+    padding-left: 15px;
+
+    /*store only, mars has a component*/
+    .f-pnav__side-info-storage-meter-bg {
+      height: 10px;
+      border-radius: 50px;
+      background-color: $color-gray-300;
+      margin-top: 4px;
+      margin-bottom: 8px;
+      width: 100%;
+    }
+
+    .f-pnav__side-info-storage-meter-percent {
+      height: 100%;
+      border-radius: 50px;
+    }
+    /* store only end*/
+
+  }
+
+  .f-pnav__side-info-club {
+    margin-top: 30px;
+    line-height: 1;
+    text-align: center;
+  }
+
+}
+
+.f-pnav__side-section-ecom-list {
+  .f-pnav__side-ecom-deals-link {
+    @include deals_link_base();
+  }
+}
+
+.f-pnav__side-section-logged-out {
+  .f-pnav__side-logged-out-button {
+    margin-right: 20px;
+    margin-top: 10px;
+    display: block;
+  }
+}
+
+
+@media (min-width: $screen-xs) {
+
+  .f-pnav {
+    padding: 0 15px;
+  }
+
+  .f-pnav__logo-image {
+    height: 28px;
+    max-width: 175px;
+  }
+
+  .f-pnav__icon-item {
+    font-size: $font-size-500;
+  }
+
+  .f-pnav__user-item-avatar {
+    visibility: visible;
+    width: auto;
+  }
+
+  .f-pnav__side.is-onscreen {
+    width: $f-pnav__side--width;
+  }
+}
+
+@media (min-width: $screen-md) {
+
+  .f-pnav__ecom-item {
+    display: flex;
+  }
+
+  .f-pnav__user-item-logged-out {
+    display: flex;
+  }
+
+  .f-pnav__side-section-ecom-list {
+    display: none;
+  }
+
+  .f-pnav__user-item-side-toggle {
+    &.is-unauthenticated {
+      display: none;
+    }
+  }
+}
+
+@media (min-width: $screen-lg) {
+  .f-pnav__ecom-link {
+    padding-left: 8px;
+    padding-right: 8px;
+    font-size: $font-size-300;
+  }
+}
+
+@media (max-width: $screen-lg-max) and (min-width: $screen-md_to_lg) {
+  .f-pnav__ecom-link {
+    font-size: $font-size-250;
+  }
+}
+
+@media (max-width: $screen-md_to_lg) and (min-width: $screen-md) {
+  .f-pnav__logo-image {
+    height: 22px;
+    max-width: 138px;
```

Comment:
this is no different than the value listed above outside the media query for this class.

Reply:
Yeah, this is tricky because it is in an in-between breakpoint - can break those out into variables, just need to come up with a good name for them. :)

Reply:
alternatively you could account for the space you need at this breakpoint elsewhere in the nav?